be/src/runtime/types.h

```diff
@@ -70,6 +70,13 @@ struct TypeDescriptor {
 
     explicit TypeDescriptor(PrimitiveType type) : type(type), len(-1), precision(-1), scale(-1) {}
 
+    bool is_wildcard_decimal() const {
```

Comment:
needn’t handle TYPE_DECIMAL.

Reply:
This method never used, I will remove it